src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1383,6 +1383,41 @@ def load_elliptic_curve_public_numbers(self, numbers):
 
         return _EllipticCurvePublicKey(self, ec_cdata, evp_pkey)
 
+    def uncompress_elliptic_curve_bytes(self, curve, point_bytes):
+        with self._tmp_bn_ctx() as bn_ctx:
+            curve_nid = self._elliptic_curve_to_nid(curve)
+
+            group = self._lib.EC_GROUP_new_by_curve_name(curve_nid)
```

Comment:
If the provided curve is unsupported by OpenSSL this will return NULL so we need to handle that case with `self.elliptic_curve_supported`. Unfortunately `_elliptic_curve_to_nid` properly errors if the NID itself isn't present, but there exist OpenSSLs where the NID exists and OpenSSL can't create the EC_GROUP

Reply:
Quick followup: Covering this new branch might be difficult actually as I'm not sure we test any OpenSSLs this stupid any more. If you can't find a way to trigger it adding an `openssl_assert` that it's not null is also a reasonable thing to do.